docs/README.md

```diff
@@ -1,9 +1,132 @@
-<!-- Front page of web site -->
+<!-- Front page of Github pages web site -->
 
-# Web page test
+# Text Extensions for Pandas
 
-This should be the main page of the Github Pages site.
+Make NLP data easier to understand with Pandas DataFrames.
+
+![Make NLP data easier to understand with Pandas](images/before_after.png)
+
+Text Extensions for Pandas lets you use Pandas DataFrames for representing and manipulating the complex data structures that flow through
+modern natural language processing (NLP) applications.
+
+We provide **Pandas extension types** to cover common data structures for representing natural language data, plus **library integrations** that turn the  coutputs of popular NLP libraries into easy-to-understand DataFrames.
+
+Want to try it out? Just type
```

Comment:
```suggestion
Want to try it out? Just type the following command in your terminal:
```

Reply:
What about "Want to try it out? Use the following command to install:"

Reply:
Fixed now.